firestore-bigquery-export/functions/src/index.ts

```diff
@@ -70,3 +78,60 @@ exports.fsexportbigquery = functions.firestore
       logs.error(err);
     }
   });
+
+exports.fsimportexistingdocs = functions.tasks
+  .taskQueue()
+  .onDispatch(async (data) => {
+    const runtime = getExtensions().runtime();
+    if (!config.doBackfill) {
+      await runtime.setProcessingState(
+        "PROCESSING_COMPLETE",
+        "No existing documents imported into BigQuery."
+      );
+      return;
+    }
+    const offset = (data["offset"] as number) ?? 0;
```

Comment:
We recommend only updating RuntimeStatus once all work is done, not every time a chunk of work is processed. We can change the wording on this message tho, if it would be clearer